content/doc/book/pipeline/overview.adoc

```diff
@@ -1,4 +1,297 @@
 ---
-layout: refresh
-refresh_to_post_id: '/doc/book/pipeline/getting-started'
+layout: section
 ---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+
+= Pipeline Overview
+
+This section of the handbook will walk through all aspects of creating a Jenkins
```

Comment:
Better present tense than future tense. "This section explains..." or "This section walks through" ("explains" is less colloquial) rather than "will walk through..."